docs/remote.md

```diff
@@ -0,0 +1,181 @@
+# Setting a Remote Environment <!-- omit in toc -->
+
+MLRun allows you to use write your code on a local machine while running your functions on a remote cluster. This tutorial explains how you can set this up.
+
+- [Prerequisites](#prerequisites)
+- [Configure Remote Environment](#configure-remote-environment)
+  - [Set Environment Variables](#set-environment-variables)
+- [IDE Configuration](#ide-configuration)
+- [Remote Environment from PyCharm](#remote-environment-from-pycharm)
+- [Remote Environment from VSCode](#remote-environment-from-vscode)
+  - [Create Environment File](#create-environment-file)
+  - [Create Python Debug Configuration](#create-python-debug-configuration)
+  - [Set Environment File in Debug Configuration](#set-environment-file-in-debug-configuration)
+- [Set Environment Variables in a Terminal](#set-environment-variables-in-a-terminal)
+
+<a id="prerequisites"></a>
+## Prerequisites
+
+Before you begin, ensure that the following prerequisites are met:
+
+1. Install MLRun locally.
+
+    You need to install MLRun locally and make sure the that the MLRun version you install is the same as the MLRun service version. Install a specific version using the following command; replace the `<version>`  placeholder with the MLRun version number (e.g., `0.5.4`):
+ 
+    ```sh
+    pip install mlrun==<version>
+    ```
+
+    If you already installed a previous version of MLRun you should first uninstall it by running:
+
+    ```sh
+    pip uninstall -y mlrun
+    ```
+
+2. Ensure that you have remote access to your MLRun service (i.e., to the service's NodePort on the remote Kubernetes cluster).
```

Comment:
In our systems the service is not exposed through NodePort but through Ingress (i.e. URL) writing NodePort might be confusing